pyaedt/application/Analysis.py

```diff
@@ -285,7 +285,8 @@ def active_setup(self):
     def active_setup(self, setup_name):
         setup_list = self.existing_analysis_setups
         if setup_list:
-            assert setup_name in setup_list, "Invalid setup name {}".format(setup_name)
+            if setup_name not in setup_list:
+                raise ValueError("Invalid setup name {}".format(setup_name))
```

Comment:
Message strings must have a vern and concluding period. Rephrase: "Setup name {}".format(setup_name) is invalid.")